EXPLAINER.md

```diff
@@ -89,14 +89,14 @@ A crash report indicates that the user was unable to continue using the page bec
   "age": 10,
   "url": "https://example.com/",
   "body": {
-    "crash_id": "c2dd3217-24f5-4bee-b74d-99bd055e7edb",
+    "crashId": "c2dd3217-24f5-4bee-b74d-99bd055e7edb",
     "type": "oom"
   }
 }
 ```
 
 The report has the following properties:
-- `crash_id`: A unique identifier. This identifier will not be meaningful to developers directly, but it can potentially be supplied to the browser vendor for more details.
+- `crashId`: A unique identifier. This identifier will not be meaningful to developers directly, but it can potentially be supplied to the browser vendor for more details.
```

Comment:
Do the TAG guidelines suggest `Id` or `ID` for abbreviations?

Reply:
Looks like they want Id:

"Id, except when the first word in a method or property"